src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta

```diff
@@ -14,15 +14,15 @@
          * @@type {{name: string, element: undefined, init: init}}
          */
         var anchor = {
-            name: "autocomplete-anchor",
+            selector: "#autocomplete-anchor",
```

Comment:
this would be a breaking change for us

Reply:
Whats your proposal to make it non breaking? The previous solution just isn't flexible enough for usage in all products